ci-operator/step-registry/hypershift/install/hypershift-install-commands.sh

```diff
@@ -41,6 +41,16 @@ if [ "${CLOUD_PROVIDER}" == "AWS" ]; then
   --external-dns-domain-filter=service.ci.hypershift.devcluster.openshift.com \
   --wait-until-available \
   ${EXTRA_ARGS}
+elif [ "${CLOUD_PROVIDER}" == "Azure" ]; then
+  "${HCP_CLI}" install --hypershift-image="${OPERATOR_IMAGE}" \
+  --managed-service ARO-HCP \
+  --external-dns-provider=azure \
+  --external-dns-credentials=etc/hypershift-aks-e2e-dns-credentials/credentials.json \
```

Comment:
this seems to be missing / before etc
Why aks? can't we just use the default infra creds for azure?